### PR TITLE
KW is a PE not just a KDAC.

### LINKED ISSUE #251: Revisit introduction.md. KDAC -> Policy Engine.
'Kubernetes Dynamic Admission Controller' is no longer an adequate description.

Kubewarden is now better described as a 'Policy Engine'. The term will need a little exposition.

A necessary update that has arisen from working on the Rancher Prime integration document with Victor.

### PR DESCRIPTION
**Pull request by jhkrug**
Will close #251 if/when merged.

I need a little more guidance for this one. From @viccuad . Thanks.

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -6,9 +6,13 @@ description: Introducing Kubewarden, a CNCF Sandbox project.
...
-Kubewarden is a [Kubernetes Dynamic Admission
-Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
-that validates incoming requests against WebAssembly policies. The policies can be developed in any programming language that generates WebAssembly binaries.
+# Introduction
+
+Kubewarden is a Kubernetes Policy Engine.
+It performs the tasks of a [Kubernetes Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+that validates incoming requests using WebAssembly policies.
+Kubewarden also provides an audit scanner to actively check policy enforcement over time.
+You can develop policies in any programming language that generates WebAssembly binaries.
```
[the remark this answers is not in the document]
Comment (viccuad) [reply docs/introduction.md:15]: in the PE engine topic, is a fuzzy definition, more like a buzzword, but nevertheless it is one that users expect to mentally classify the project.

[the remark this answers is not in the document]
Comment (jhkrug) [reply docs/introduction.md]: Thanks, I made some changes.

**Review by viccuad — APPROVED** [review]

File: docs/introduction.md [hunk 2]
```diff
@@ -6,9 +6,19 @@ description: Introducing Kubewarden, a CNCF Sandbox project.
 keywords: [kubewarden, cncf, cncf sandbox, kubernetes]
 ---
 
-Kubewarden is a [Kubernetes Dynamic Admission
-Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
-that validates incoming requests against WebAssembly policies. The policies can be developed in any programming language that generates WebAssembly binaries.
+# Introduction
+
+Kubewarden is a Kubernetes Policy Engine.
+It aims to be the Universal Policy Engine for Kubernetes
```
Comment (flavio) [inline docs/introduction.md:12]:
```suggestion
It aims to be the Universal Policy Engine for Kubernetes.
```

File: docs/introduction.md [hunk 3]
```diff
@@ -6,9 +6,19 @@ description: Introducing Kubewarden, a CNCF Sandbox project.
...
+
+You can reuse policies from other policy engines without writing any code.
```
Comment (flavio) [inline docs/introduction.md:14]:
```suggestion
You can reuse policies from other policy engines without having to rewrite them.
```

**Review by flavio — COMMENTED** [review]
I made some suggestions

File: docs/introduction.md (hunk 2 above)
Comment (jhkrug) [reply docs/introduction.md:12]: Thanks.

**Pull request merged**